.github/workflows/python.yml

```diff
@@ -105,15 +105,20 @@ jobs:
 
     runs-on: ${{ matrix.runs_on }}
 
+    container: ${{ matrix.container }}
```

Comment:
Oh that's cool that this falls through to "no container" if the value doesn't exist in the matrix